src/kit.ts

```diff
@@ -549,14 +549,29 @@ async function collectDevBatVars(devbat: string, args: string[], major_version:
   for (const envvar of MSVC_ENVIRONMENT_VARIABLES) {
     bat.push(`echo ${envvar} := %${envvar}% >> ${envfname}`);
   }
-  const batContent = bat.join('\r\n');
-  const batpath = path.join(paths.tmpDir, batfname);
-  const envpath = path.join(paths.tmpDir, envfname);
+
+  // writeFile and unlink don't need quotes (they work just fine with an unquoted path with space)
+  // but they might fail sometimes if quotes are present, so remove for now any quotes
+  // that may have been defined by the user (the command prompt experience makes it very likely
+  // for the user to use quotes when defining an environment variable with a space containing path).
+  const tmpDir: string = paths.tmpDir?.replace(/"/mg, "");
```

Comment:
I think we should only trim quotes at the beginning or end of the string.  Linux lets you put quotes in folder and file names.

Reply:
Oh, that's right, good catch, thank you.